azext_hanaonazure/custom.py

```diff
@@ -280,10 +280,17 @@ def create_providerinstance(
       monitor_name,
       provider_instance_name,
       provider_instance_type,
-      provider_instance_properties):
+      provider_instance_properties,
+      provider_instance_metadata=None):
```

Comment:
What about updating the caller where this function gets called?

Reply:
The framework magically call this function and can detect the parameters and their default.
Just updating the signature of this method is enough to change the command. I know, magic